src/main/drivers/at32/serial_uart_at32f43x.c

```diff
@@ -387,12 +387,21 @@ uartPort_t *serialUART(UARTDevice_e device, uint32_t baudRate, portMode_e mode,
 
     if ((options & SERIAL_BIDIR) && txIO) {
         //mode,speed,otype,pupd
+#ifndef USE_NONCOMPIANT_SMARTAUDIO
```

Comment:
this way SA problems are leaking into other code. It is better to adjust argument in SA related code. 

But as maintenance ..  I hope this will become dark history soon

Reply:
Note in addition `USE_AKK_SMARTAUDIO` is needed for the holybro device too https://holybro.com/products/atlatl-hv-micro

Asked mfg to check using `USE_AKK_SMARTAUDIO` instead of `USE_NONCOMPLIANT_SMARTAUDIO` - BUT as we already have added `USE_SMARTAUDIO_NOPULLDOWN` in #13797  --> we might could use that instead of adding `USE_NONCOMPLIANT_SMARTAUDIO` ?

Reply:
Note in addition `USE_AKK_SMARTAUDIO` is needed for the holybro device too https://holybro.com/products/atlatl-hv-micro on a prototype AT32.

Could we use `USE_SMARTAUDIO_NOPULLDOWN` instead of `USE_NONCOMPLIANT_SMARTAUDIO` - as added ` in #13797 or some similar define if this a different situation ???